app/api/helpers/mail.py

```diff
@@ -543,6 +546,13 @@ def send_email_to_attendees(order):
         order_view_url=order.site_view_link,
     )
 
+    event_date = event.starts_at.strftime('%d %B %Y')
+    event_time = (
+        event.starts_at.replace(tzinfo=pytz.timezone('UTC'))
+        .astimezone(pytz.timezone(event.timezone))
+        .strftime('%H:%M (%Z%z)')
+    )
+
```

Comment:
at one place you have used `.replace(tzinfo=pytz.timezone('UTC'))
        .astimezone(pytz.timezone(event.timezone))` and other not. Any specific reason?

Reply:
Which other place you are talking about??

If you are asking about why I have changed only 1 attendee email, @mariobehling asked me to do so.

Reply:
Well, I think the event time and time zone should be handled in the same way in both emails. But, lets try to get together in the meeting so we are all on the same page. @progmatic-99 Could you open another issue for the second email please. So, we can finalize this PR? Thanks.

Reply:
Ok @mariobehling